coding-concepts.js

```diff
@@ -15,45 +15,45 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2022"
+// const cohort = "LEARN 2022"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: i believe this will determine the length of "LEARN 2022"
+// b) Verify and explain: i was correct, the outcome was the length of 10. this is because .length returns the numer of elements in an array
 
 
-// --------------------2) What will this log?
+// // --------------------2) What will this log?
 
-const greeting = "Hello World!"
+// const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: i believe this will give the outcome of "o" since that is the index of [4]
+// // b) Verify and explain: i was correct, this is because bracket notations allow us to access elements by index
 
 
-// --------------------3) What will this log?
+// // --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
+// const languages = ["JavaScript", "Ruby", "Python", "C++"]
+// const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: i believe the outcome will be Ruby, considering Ruby is in the first index
+// // b) Verify and explain: i was correct. if you set const index to 0, 1, 2, or 3 the outcome will be whatever value is in that index
 
 
-// --------------------4) What will this log?
+// // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
+// const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: i beleieve .toUpperCase will return the words in all uppercase letters
+// // b) Verify and explain: i recieved an error message because "toUpperCase" is not a function
 
 
-// --------------------5) What will this log?
+// // --------------------5) What will this log?
 
-const dataTypes = ["number", "string", "Boolean"]
-// console.log(typeof dataTypes.length)
+// const dataTypes = ["number", "string", "Boolean"]
+// console.log(dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: this will return the length of this array. 20
+// // b) Verify and explain: the outcome was "number" the original console.log(type of datatTypes.length) returned the word "number". i took out "typeOf" (assuming it was a typo, maybe from me) and the outcome was "3". this is the number of elements in the array. so .length returns the number of elements, not the number of individula letters.
```

Comment:
typeof was not a typo.  In this case when it returned 'number' initially, it was giving you the data type of what follows it which was dataTypes.length.